HopsanGUI/GraphicsView.cpp

```diff
@@ -660,6 +703,30 @@ void GraphicsView::mousePressEvent(QMouseEvent *event)
 
 void GraphicsView::mouseReleaseEvent(QMouseEvent *event)
 {
+    if(mIgnoreNextMouseReleaseEvent) {
+        QGraphicsView::mouseReleaseEvent(event);
+        mIgnoreNextMouseReleaseEvent = false;
+        return;
+    }
+    if(mpAddComponentLineEdit->isVisible())
+    {
+        mpAddComponentLineEdit->hide();
+    }
+    else if(!mpContainerObject->isCreatingConnector() &&
+            mpContainerObject->getSelectedModelObjectPtrs().isEmpty() &&
+            this->rubberBandRect().isEmpty())
+    {
+        QCursor cursor;
+        QPointF pos = mapFromGlobal(cursor.pos());
+        QRect tempRect(pos.x(), pos.y(),300,1);
+
+        QStringList typeNames = QStringList() << gpLibraryHandler->getLoadedTypeNames();
+        mpAddComponentLineEdit->completer()->setModel(new QStringListModel(typeNames,mpAddComponentLineEdit));
+        mpAddComponentLineEdit->setGeometry(pos.x(), pos.y(), 200,30);
```

Comment:
I have no idea how to solve this, but hard-coding sizes in pixels may not work well on monitors with some other resolution. No need to fix this now, maybe deserve an issue of its own. Problem exist in many palces in Hopsan

Reply:
I agree, but I'm not sure if it's possible. Perhaps you could look at the size of the default font or something.